apps/server/src/modules/common-cartridge/controller/dto/course-export-body.response.ts

```diff
@@ -0,0 +1,12 @@
+import { CourseCommonCartridgeMetadataDto } from '../../common-cartridge-client/dto/course-common-cartridge-metadata.dto';
+import { CourseFileIdsResponse } from './common-cartridge.response';
+
+export class CourseExportBodyResponse {
+	courseFileIds!: CourseFileIdsResponse;
+
+	courseCommonCartridgeMetadata!: CourseCommonCartridgeMetadataDto;
+
+	constructor(courseExportBodyResponse: CourseExportBodyResponse) {
+		Object.assign(this, courseExportBodyResponse);
```

Comment:
Suggestion: I would prefer an explicit initialization here. Then it is clearer which properties are expected, it has better type safety and enables better debugging.


```suggestion
	export class CourseExportBodyResponse {
        courseFileIds: CourseFileIdsResponse;
        courseCommonCartridgeMetadata: CourseCommonCartridgeMetadataDto;

         constructor(courseExportBodyResponse: Partial<CourseExportBodyResponse>) {
           this.courseFileIds = courseExportBodyResponse.courseFileIds!;
          this.courseCommonCartridgeMetadata = courseExportBodyResponse.courseCommonCartridgeMetadata!;
         }
    }
```

Reply:
done ✅